src/core1_task.rs

```diff
@@ -621,13 +784,13 @@ pub fn core_1_task(
                         making_status_recording = true;
                     }
                 } else {
-                    if let Some((_, window_end)) = &current_recording_window {
-                        if synced_date_time.date_time_utc + Duration::minutes(1) > *window_end {
-                            warn!("Make shutdown status recording");
-                            should_start_new_recording = true;
-                            made_shutdown_status_recording = true;
-                            making_status_recording = true;
-                        }
+                    if synced_date_time.date_time_utc + Duration::minutes(1)
+                        > current_recording_window.1
```

Comment:
This doesn't really feel clearer than the code it's replacing that restructures `window_end` from the tuple...
Could still be `let (_, window_end) = &current_recording_window; ...` I think?